sambaplayersdk/src/main/java/com/sambatech/player/SambaPlayer.java

```diff
@@ -364,9 +365,16 @@ public void onConnected(final CastSession castSession) {
             }
 
 
-            if (media.drmRequest != null)
-                castObject.setDrm(new CastDRM(media.drmRequest.getLicenseParam("SessionId"),
-                        media.drmRequest.getLicenseParam("Ticket")));
+            if (media.drmRequest != null) {
+                CastDRM castDRM = new CastDRM(media.drmRequest.getLicenseParam("SessionId"),
+                        media.drmRequest.getLicenseParam("Ticket"));
+                if (media.drmRequest.getProvider() != null && media.drmRequest.getProvider().equals("SAMBA_DRM") && media.drmRequest.getToken() != null) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "SAMBA_DRM" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1132)